OracleWebCenterSites/README.md

```diff
@@ -291,11 +291,11 @@ a. To run WebLogic Managed Server container, go to folder where `wcsitesserver.e
  
 b. Run the following command and pass the environment file name as a parameter: 
 ```
-   $ docker run -d -t --name <container_name> --network=<network_name> --volumes-from <admin_container_name> -p <sites_port>:7002 -p <sites_ssl_port>:9002 --env-file <environment_file> <repo_name:tag_name> /bin/bash -c "/u01/oracle/sites-container-scripts/startSitesServer.sh; /bin/bash"
+   $ docker run -d -t --name <container_name> --network=<network_name> --volumes-from <admin_container_name> -p <sites_port>:7002 -p <sites_ssl_port>:9002 --env-file <environment_file> <repo_name:tag_name> sites-container-scripts/startSitesServer.sh
```

Comment:
Why does a user need to specify `sites-container-scripts/startSitesServer.sh` on the command line? Why doesn't the container just run this automatically?